docs/src/main/asciidoc/update-to-quarkus-3.adoc

```diff
@@ -0,0 +1,89 @@
+////
+This document is maintained in the main Quarkus repository, and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="update-projects-to-quarkus-3-howto"]
+= Update projects to Quarkus 3 automatically
+:extension-status: preview
```

Comment:
The order and ID items in the header items differ from the [contributor guide template & example](https://quarkus.io/guides/doc-contribute-docs-howto#create-quarkus-content-in-asciidoc) for the new diataxis implementation.
 
Minor observation, not sure it makes a difference in upstream output, but it might for our downstream post-fix script/processes. The more consistent the easier for later.

Reply:
Updated. Thanks.